loopdb/sqlite.go

```diff
@@ -211,14 +211,19 @@ func (db *BaseDB) ExecTx(ctx context.Context, txOptions TxOptions,
 // FixFaultyTimestamps fixes faulty timestamps in the database, caused
 // by using milliseconds instead of seconds as the publication deadline.
 func (b *BaseDB) FixFaultyTimestamps(ctx context.Context) error {
-
 	// Manually fetch all the loop out swaps.
 	rows, err := b.DB.QueryContext(
 		ctx, "SELECT swap_hash, publication_deadline FROM loopout_swaps",
 	)
 	if err != nil {
 		return err
 	}
+	if rows.Err() != nil {
```

Comment:
Nice catch! I believe instead what we'd want is first go through the rows (line 237), then after that `for` loop we'd:
```
	if err := rows.Close(); err != nil {
		return nil, err
	}
	if err := rows.Err(); err != nil {
		return nil, err
	}
```

Reply:
Ah, good point. Though the linter really wants me to call `Close` with a `defer`... So how does that latest version look to you?

Reply:
Looks good!